server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -119,6 +122,124 @@ const orderMutations = {
       return models.Order.findOne(query);
     },
   },
+  updateOrder: {
+    type: Order,
+    description: "Update an Order's amount, tier, payment method, or frequency",
+    args: {
+      order: {
+        type: new GraphQLNonNull(OrderReferenceInput),
+        description: 'Object matching the OrderReferenceInput (id)',
+      },
+      amount: {
+        type: GraphQLInt,
+        description: 'Amount in cents of the order',
+      },
+      frequency: {
+        type: GraphQLString,
+        description: 'Frequency of the recurring order, either MONTHLY or YEARLY',
+      },
+      tier: {
+        type: GraphQLString,
+        description: 'The tier of the recurring contribution, like Backer or Sponsor',
+      },
+      paymentMethod: {
+        type: PaymentMethodReferenceInput,
+        description: 'Object matching the PaymentMethodReferenceInput (uuid)',
+      },
+    },
+    async resolve(_, args, req) {
+      const decodedId = getDecodedId(args.order.id);
+
+      if (!req.remoteUser) {
+        throw new Unauthorized('You need to be logged in to update a subscription');
+      }
+
+      const { paymentMethod } = args;
+
+      const query = {
+        where: {
+          id: decodedId,
+        },
+        include: [{ model: models.Subscription }, { model: models.PaymentMethod, as: 'paymentMethod' }],
```

Comment:
Are you planning to do something with the paymentMethod relation?

Reply:
Not that I can tell yet. Will remove.